.github/workflows/runtime_ci.yml

```diff
@@ -0,0 +1,68 @@
+name: Runtime Testing
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test_runtime:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
+        python-version: ["3.10"]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y hdf5-tools libhdf5-openmpi-dev openmpi-bin
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade pytest mpi4py
+          # the organization of HDF5 files from apt isn't
+          # ideal for usage with darshan, but we can cheat
+          # a little
+          sudo cp /usr/include/hdf5/openmpi/*.h /usr/include
```

Comment:
you don't have to change this, but i confirmed you can work around the header location issue above by just adding `/usr/include/hdf5/openmpi/` to C_INCLUDE_PATH, i.e.,:

`export C_INCLUDE_PATH=$C_INCLUDE_PATH:/usr/include/hdf5/openmpi/`

in the "Install darshan-runtime step". Still an annoying additional step, but maybe less hack-y.